runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -128,6 +146,34 @@ private boolean isInKingdom()
 			&& KINGDOM_REGION.contains(client.getLocalPlayer().getWorldLocation().getRegionID());
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.KINGDOM_GROUP_ID && config.showKingdomValue())
```

Comment:
Use a guard clause instead of a large `if` statement, like so:

```java
if (event.getGroup() != WidgetID.KINGDOM_GROUP_ID || !config.showKingdomValue())
{
	return;
}

// rest of code goes here
```